cms/djangoapps/contentstore/views/tests/test_import_export.py

```diff
@@ -686,6 +687,294 @@ def test_import_status_response_is_not_cached(self):
         self.assertEqual(resp.headers['Cache-Control'], 'no-cache, no-store, must-revalidate')
 
 
+@ddt.ddt
+@override_settings(CONTENTSTORE=TEST_DATA_CONTENTSTORE)
+class ImportZipTestCase(CourseTestCase):
```

Comment:
I see a lot of duplication here with the ImportTests for tar. Perhaps you could just create a mixin with all the common functionality, or just use DDT to run both zip and tar tests in the original class.

Reply:
done